namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-domain-event-logger.tf

```diff
@@ -0,0 +1,118 @@
+resource "kubernetes_secret" "hmpps-domain-event-logger" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = "hmpps-domain-event-logger-dev"
+  }
+
+  data = {
+    access_key_id     = module.hmpps-domain-events.access_key_id
+    secret_access_key = module.hmpps-domain-events.secret_access_key
+    topic_arn         = module.hmpps-domain-events.topic_arn
+  }
+}
+
+module "hmpps_domain_event_logger_queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.8"
```

Comment:
wasn't sure whether to use 4.9 - can update if required

Reply:
We have a 4.9.1, best to use latest release.